docs/logging.md

```diff
@@ -14,8 +14,8 @@ The default locations for worker logs are:
 
 | Platform | Default worker log path |
 | --- | --- |
-| Linux | `/var/log/amazon/deadline/logs` |
-| MacOS | `/var/log/amazon/deadline/logs` |
+| Linux | `/var/log/amazon/deadline` |
+| MacOS | `/var/log/amazon/deadline` |
```

Comment:
why does Windows have a Logs folder but the other two don't? Should we keep consistent for them?

Reply:
It's under `/var/log` on posix-based systems. That's universally understood to be for logging files.